redaxo/src/core/lib/api_function.php

```diff
@@ -328,6 +340,9 @@ public function isSuccessfull()
         return $this->succeeded;
     }
 
+    /**
+     * @return false|string
```

Comment:
```suggestion
     * @return string
```

(Hier ist auch so ein Fall)